lib/check_params.h

```diff
@@ -878,9 +878,11 @@ void printQudaMultigridParam(QudaMultigridParam *param) {
     P(setup_use_mma[i], QUDA_BOOLEAN_FALSE);
 #endif
     P(dslash_use_mma[i], QUDA_BOOLEAN_FALSE);
+    P(transfer_use_mma[i], QUDA_BOOLEAN_FALSE);
```

Comment:
Meta comment: while eventually we will want the mma options to be default on, I think for now we should leave them opt in.  Switching to default on will come when I've got the entire coarse grid correction working in MMA order, avoiding the reordering overheads.

Reply:
For now only setup has the MMA option default to on.